modules/mod_acl_user_groups/mod_acl_user_groups.erl

```diff
@@ -136,6 +137,7 @@ ug_delete(Ids, Context) ->
 
     end.
 
+-spec ug_move_and_delete([pos_integer()], m_rsc:resource_id(), #context{}) -> ok.
 ug_move_and_delete(Ids, ToGroupId, Context) ->
     z_session_page:add_script(z_render:wire({mask, [{message, ?__("Deleting...", Context)}]}, Context)),
```

Comment:
Line 142 is too long:     z_session_page:add_script(z_render:wire({mask, [{message, ?__("Deleting...", Context)}]}, Context)),.